### DISCUSSION TITLE
Entrega actividad 3 - Grupo 10

### DISCUSSION THREAD

**Discussion by Nachoff13** [frlp-utn-ingsoft/catedra#140] [Show and tell]
https://github.com/MilagrosSoberon/vetsoft

**Comment by ivanduva** [comment]
Buenas,
La actividad está aprobada. Lo único para comentar es que no es necesario hacer un PR separado para los tests, pero igualmente vemos que lo corrigieron en la siguiente actividad.

Saludos
